src/test/java/org/broadinstitute/hellbender/MainTest.java

```diff
@@ -16,4 +20,14 @@ public void testPrintReadsVersion() {
         String out = BaseTest.captureStderr(() -> Main.main(new String[]{"PrintReads", "--version"}));
         BaseTest.assertContains(out, "Version:");
     }
+
+    @Test
+    public void testCustomLogLevel() {
+        // This first assert only serves to validate the test conditions and ensure
+        // that the rest of the test is valid.
+        Assert.assertTrue(logger.getLevel() != Level.DEBUG);
+        final File testFile = new File(getTestDataDir(), "print_reads.sam");
+        Main.main(new String[]{"CountReads", "-I", testFile.getPath(), "--VERBOSITY", "DEBUG"});
+        Assert.assertTrue(logger.getLevel() == Level.DEBUG);
```

Comment:
So this is setting the global logger, but I'm not sure what that means for the log level of the rest of the tests.  Would it be possible to capture the initial logging level and the reset it after the test?